.eslintrc.json

```diff
@@ -1,17 +1,35 @@
 {
   "env": {
     "browser": true,
-    "es2021": true,
-    "node": true
+    "es2020": true,
```

Comment:
es2021 is broken.

https://github.com/NullVoxPopuli/eslint-plugin-decorator-position/issues/138